tests/sentry/auth/test_access.py

```diff
@@ -541,23 +541,55 @@ def test_superuser(self):
         assert result.has_permission("test.permission")
 
     @patch("sentry.auth.access.is_active_superuser", return_value=True)
-    def test_superuser_in_organization_scopes(self, mock_is_active_superuser):
+    def test_superuser_scopes(self, mock_is_active_superuser):
+        # superuser not in organization
         request = self.make_request(user=self.superuser)
 
         # needs org in request in order to assign any scopes
         result = self.from_request(request, self.org)
         assert result.scopes == SUPERUSER_SCOPES
 
+        # superuser in organization
+        self.create_member(user=self.superuser, organization=self.org, role="member")
+
+        result = self.from_request(request, self.org)
+        assert result.scopes == SUPERUSER_SCOPES
+
     @with_feature("auth:enterprise-superuser-read-write")
     @override_settings(SENTRY_SELF_HOSTED=False)
     @patch("sentry.auth.access.is_active_superuser", return_value=True)
-    def test_superuser_in_organization_readonly_scopes(self, mock_is_active_superuser):
+    def test_superuser_readonly_scopes(self, mock_is_active_superuser):
```

Comment:
i have two cases in these tests now. i wasn't passing the correct thing into the feature check, but it returned `True` anyway.

i verified that these generate separate cases where `member = None` and `member = OrganizationMember`. we were failing before in the case where `member = None` when you weren't part of the org since we were passing `None` into the feature flag check